content/docs/100-get-started/index.mdx

```diff
@@ -1,8 +1,6 @@
 ---
 title: Get Started
-nav_title: Get Started
 show_child_cards: true
-excerpt: Raydiant makes it easy to build apps for digital signage and kiosks.
```

Comment:
Top-level docs don't need excerpts as the card summaries are only available for sub-sections.